### PR TITLE
perf(📦): reduce alert css size

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [x] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/25892

### 💡 Background and solution

使用 flex 对齐，简化了一些样式代码。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    -       |
| 🇨🇳 Chinese |   -        |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
来个 approve

**Review by xrkffgg — APPROVED** [review]

**Comment by jesse3mh9a** [comment]
@afc163  `rtl`还有些未调整

**Comment by xrkffgg** [comment]
> @afc163  `rtl`还有些未调整

再发pr，修正吧

File: components/alert/style/index.less [hunk 1]
```diff
@@ -111,28 +102,18 @@
   }
 
   &-with-description {
-    position: relative;
+    align-items: flex-start;
     padding: @alert-with-description-padding;
-    color: @alert-text-color;
```
Comment (mrlubos) [inline components/alert/style/index.less:116]: @afc163 為什麼你把這行刪掉？謝謝

**Pull request merged**